### PR TITLE
Use namespace for events

### PR DESCRIPTION
**Pull request by Indigo744**
Implements #144 

How to check in browser:
```javascript
$._data( $(".map")[0], "events" )["resizeEnd"][0]; // the first resizeEnd event object
$._data( $(".map")[0], "events" )["resizeEnd"][0].type; // return 'resizeEnd'
$._data( $(".map")[0], "events" )["resizeEnd"][0].namespace; // return 'mapael'
```

### CODE REVIEW

File: js/jquery.mapael.js [hunk 1]
```diff
@@ -673,12 +673,12 @@
                 $map.unbind("resizeEnd", createPlotLegend);
```
Comment (neveldo) [inline js/jquery.mapael.js]: Shouldn't be replaced by `$map.unbind("resizeEnd." + pluginName, createPlotLegend);` instead ?

Comment (Indigo744) [inline js/jquery.mapael.js]: You are right. I changed only for event binding. I will correct it asap!

**Comment by Indigo744** [comment]
MMh, I forgot also to change the trigger part. Wait before merging! ;-)

**Comment by Indigo744** [comment]
Good to go :-)

**Comment by neveldo** [comment]
Great, thank you !

**Pull request merged**